resources/stakeholder_workshops/user_research_questions.md

```diff
@@ -0,0 +1,53 @@
+# Process & Guidance for Designing Effective User Research Questions
+
+## Pre-work before drafting usability questions
+
+- With team, specify features of MVP
+- With team, specify Experiment / Hypotheses around current MVP
+- With team draft user journeys
+- Identify gaps in understanding of user journey
+
+## Decide
+
```

Comment:
- What features or aspects of the MVP are you planning to test?
- What decisions will be made from the outcome of the user research?